Cargo.toml

```diff
@@ -56,13 +51,16 @@ memchr = "2.3"
 uuid = { version = "1.0", optional = true }
 smallvec = "1.6.1"
 cast = { version = "0.3", features = ["std"] }
-arrow = { version = "40", default-features = false, features = ["prettyprint", "ffi"] }
+arrow = { version = "39.0.0", default-features = false, features = ["prettyprint", "ffi"] }
```

Comment:
DependencyBot will complain about this.

Does polars/arrows update their dependency regularly?

Reply:
Yes, but there's still a bit delay. 
I see that `arrow` regularly update their version.
If we want to keep the `arrow` updated, we have to make PRs in `arrow2`, `polars` to update their `arrow` version.